subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -1126,7 +1126,7 @@ config BT_CTLR_FORCE_MD_AUTO
 
 config BT_CTLR_CONN_RANDOM_FORCE
 	bool "Random forced scheduling for peripheral on missed anchor point"
-	depends on BT_PERIPHERAL
+	depends on BT_PERIPHERAL && (BT_MAX_CONN > 1)
```

Comment:
This change does not seem to be related to the sample commit. Please move to separate commit or separate PR :)